.github/workflows/build-push-docker.yml

```diff
@@ -3,78 +3,17 @@ name: Build and Push Subgraph Deployer Docker Image
 on:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
-      - 'v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+'
+      - '*'
+  pull_request:
+    branches:
+      - '*'
   workflow_dispatch:
- 
+
 jobs:
-  # TODO: See how to ensure that the default CI workflow has run before this workflow
```

Comment:
Isn't this still relevant?

Reply:
We have now 2 workflows: 

- One on every pull requests
- One on tag representing the production ready